composer.json

```diff
@@ -152,7 +174,8 @@
         "drupal/views_infinite_scroll": "^2.0",
         "drupal/views_taxonomy_term_name_depth": "^7.0",
         "drupal/webp": "^1.0@beta",
-        "drupal/xmlsitemap": "^1.0",
+        "drupal/xmlsitemap": "^2.0",
+        "noli42/chosen": "3.0.0",
         "harvesthq/chosen": "^1.8",
```

Comment:
The `noli42/chosen` library is replacing this one. You'll want to remove it.

Reply:
It looks like it's still required in the current version of Stanford Profile? https://github.com/SU-SWS/stanford_profile/blob/f070578b8e6bcb2e0bb2640bfcbfec78dd02a420/composer.json#L172

Is this a transition thing?

Reply:
I don't see the `harvesthq` library there.

Reply:
Okay I see the confusion. My comment was supposed to be calling out line 179 with `harvesthq/chosen` on it but it looks like I'm calling out the diff addition of the `noli42` library.

Reply:
@joegl Gotcha. Yeah, I thought maybe you meant the module shouldn't be present. I'll take a look.